### PR TITLE
Feat/5aa reindex

### PR DESCRIPTION
**Pull request by giangbui**
At this moment, we will ignore all the 5aa bucket objects but we need to reindex them

### CODE REVIEW

File: scripts/deletion.py [hunk 1]
```diff
@@ -150,19 +151,28 @@ def _remove_object_from_s3(s3, indexclient, f, target_bucket):
         deletion_log.message = str(e)
         return deletion_log
 
-    if res["Deleted"]:
+    if res.get("Deleted"):
         try:
+            deletion_log.deleted = True
             remove_url_from_indexd_record(f.get("id"), [full_path], indexclient)
```
[the remark this answers is not in the document]
Comment (giangbui) [reply scripts/deletion.py]: It is in the right position. After we delete it from cloud, it should be marked as deleted and then remove it from indexd after that

File: scripts/deletion.py [hunk 2]
```diff
@@ -177,7 +187,11 @@ def _remove_object_from_gs(client, indexclient, f, target_bucket):
         list(DeletionLog)
 
     """
+    url = get_structure_gs_url(f, IGNORED_FILES)
```
[the remark this answers is not in the document]
Comment (giangbui) [reply scripts/deletion.py]: oh I changed the name of the function but forgot to update the code that calls to. Fixed in ff2896173f5e1a07fa76bb16664068d6b9411b94

File: scripts/deletion.py [hunk 3]
```diff
@@ -177,7 +187,12 @@ def _remove_object_from_gs(client, indexclient, f, target_bucket):
         list(DeletionLog)
 
     """
+    object_key = get_structured_object_key(f, IGNORED_FILES)
```
Comment (paulineribeyre) [inline scripts/deletion.py]: should this be `utils.get_ignored_files(IGNORED_FILES)` instead of `IGNORED_FILES`?

Comment (giangbui) [reply scripts/deletion.py]: Good catch! Thanks
fa197bd2c7f245e7a0fc3771be8044f233b9656d

**Pull request merged**